### ISSUE TITLE
NTR: Presence of bezoar, Presence of trichobezoar

### ISSUE THREAD

**Issue by LCCarmody** [mgijax/mammalian-phenotype-ontology#4114]
**Terms:** Presence of bezoar (parent)
                       Presence of trichobezoar (child)

**Definitions:**Presence of bezoar: Presence of hard, indigestible masses that form in the gastrointestinal tract, usually in the stomach.
                    Presence of trichobezoar: Presence of  a hairball made of   swallowed hair that can form in the stomach or small intestine
**Synonyms:** (bezoar) Presence of mass in gastrointestinal tract
                    (trichobezoar) Presence of a hariball in gastrointestinal tract

**Logical def suggestion:** 
(bezoar)
'has part' some 
    ('material entity'
     and ('characteristic of part of' some 'alimentary part of the gastrointestinal system')
     and ('has modifier' some abnormal))
(trichobezoar)
'has part' some
('strand of hair'
 and ('characteristic of part of' some 'alimentary part of the gastrointestinal system')
     and ('has modifier' some abnormal))

**Parent:** MP_0000462 Abnormal digestive system morphology (although, this might be a time to add 'abnormal gastrointestinal tract' and group 'gastrointestinal polyps' and 'gastrointestinal ulcer'?   Would not be opposed to that sort of hierarchy.

**Reference/Comment:** According to this paper (PMID: [24672418](https://pubmed.ncbi.nlm.nih.gov/24672418/)), there are also Lactobezoars, pharmacobezoars, phytobezoars.  We are not in need of these terms, but it may be worth considering.

**Comment:** These phenotypes may be from some sort of pica/or abnormal eating, but may be due to abnormality in gastrointestinal motility.  We would like the terms to observe the phenotypes observed and not necessarily the underlying cause.

@jmfostel

**Comment by sbello** [comment]
To keep with standard MP label structure I think I would call this 'abnormal bezoar incidence' we could add increased and increased child terms assuming bezoars are ever normally found in a mammalian species, otherwise I could just go with increased bezoar incidence

[10982771](https://pubmed.ncbi.nlm.nih.gov/10982771/) has mouse model with bezoar 
https://www.informatics.jax.org/allele/genoview/MGI:4366780

**Comment by sbello** [comment]
Poking in pubmed there are some reports that refer to bezoars in other organs. Consider call the term 'gastrointestinal bezoar' to make it clear that these are in the gastrointestinal tract

ref:
https://pubmed.ncbi.nlm.nih.gov/24290190/ https://pubmed.ncbi.nlm.nih.gov/21695474/ - renal

**Comment by jmfostel** [comment]
thank you!  abnormal gastrointestinal bezoar incidence is a good term label.  i do not know the background incidence, but imagine that it would be expected to be near zero.  thanks for this work.

**Comment by sbello** [comment]
@jmfostel do you have an ORCID that I can include in the term as a contributor?

**Comment by jmfostel** [comment]
yes, here it is: 0000-0001-5858-4393

**Comment by sbello** [comment]
Added terms
abnormal gastrointestinal bezoar incidence MP:0014661
abnormal gastrointestinal phytobezoar incidence MP:0014665
abnormal gastrointestinal pharmacobezoar incidence MP:0014664
abnormal gastrointestinal lactobezoar incidence MP:0014663
abnormal gastrointestinal trichobezoar incidence MP:0014662
